### PR TITLE
Update guide with information on MySQL and Postgres

### PR DESCRIPTION
**Pull request by pshevtsov**
This PR introduces the following changes:

- Adds supported versions of MySQL and PostgreSQL to the supported versions table
- Adds sections on creating/configuring users for MySQL and PostgreSQL
- Improves typography and formatting

### CODE REVIEW

File: pages/tidal tools/analyze_database.md [hunk 1]
```diff
@@ -19,14 +19,14 @@ It is capable of analyzing both **Oracle and SQL Server databases**. Providing a
...
-|-----------------------|
-| [Beta](mailto:info@tidalmigrations.com?subject=Interested in Oracle 8i Database Analysis) Oracle Database 8i (8.1) | SQL Server 2008R2 |
-| [Beta](mailto:info@tidalmigrations.com?subject=Interested in Oracle 9 Database Analysis) Oracle Database 9i Release 2 (9.2)   | SQL Server 2016 |
-| Oracle Database 10g Release 2 (10.2) | SQL Server 2017 |
-| Oracle Database 11g Release 1 (11.1) |
-| Oracle Database 11g Release 2 (11.2) |
-| Oracle Database 12c Release 1 (12.1) |
+| Oracle                | SQL Server | MySQL | PostgreSQL |
+|-----------------------|------------|-------|------------|
+| [Beta](mailto:info@tidalmigrations.com?subject=Interested in Oracle 8i Database Analysis) Oracle Database 8i (8.1) | SQL Server 2008R2 | 5.6 | 8 |
```
Comment (jmariomejiap) [inline pages/tidal tools/analyze_database.md:24]: @pshevtsov I think there is something off with the hyperlink here.

image

File: pages/tidal tools/analyze_database.md [hunk 2]
```diff
@@ -48,20 +48,20 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
...
-  - user - A username to authenticate with the database with, see below for more details.
-  - password - A password for the corresponding user.
-  - name - A common name for your database could be the same or different from db_name, but this value is arbitrary and only for your reference.
+  - `id` - The id of the database from your Tidal Migrations account. You can find it in the URL bar when looking at a database instance. ex. If you are viewing a database instance in Tidal Migrations, the URL will show https://demo2.tidalmg.com/#/database_instances/111 in this case 111 is the database instance ID.
+  - `engine` - The database vendor, either `Oracle` or `SQL Server`, it is not case sensitive.
+  - `host` - The hostname of the server that the database is located on and is accessible via a network connection from your current device and location.
+  - `port` - The port that the host has open and the database can accept connections on, the default for Oracle is `1521`, for SQL Server it is `1433`, for MySQL it is `3306`, and for PostgreSQL the default port is `5432`.
```
Comment (jmariomejiap) [inline pages/tidal tools/analyze_database.md:54]: nice addition. 👍

File: pages/tidal tools/analyze_database.md [hunk 3]
```diff
@@ -48,20 +48,20 @@ For example, in Oracle databases, the Data Dictionary and AWR repository tables
...
+  - `user` - A username to authenticate with the database with, see below for more details.
+  - `password` - A password for the corresponding user.
+  - `name` - A common name for your database could be the same or different from db_name, but this value is arbitrary and only for your reference.
 
 ### Oracle User
 
 You can either use the default `SYSDBA` user or you can create a user with the same set of permissions:
 
-```
+```sql
```
Comment (jmariomejiap) [inline pages/tidal tools/analyze_database.md:64]: It looks so good 😍

**Review by jmariomejiap — CHANGES_REQUESTED** [review]
It looking great. besides the formatting issue, I think it is 💯

File: pages/tidal tools/analyze_database.md (hunk 1 above)
Comment (pshevtsov) [reply pages/tidal tools/analyze_database.md:24]: It renders OK with Jekyll
image

Comment (jmariomejiap) [reply pages/tidal tools/analyze_database.md:24]: oh, great then. 👍  🚀

**Review by jmariomejiap — APPROVED** [review]
🚀

**Comment by prees1** [comment]
Looks great! 👍 

There are several references to `Oracle and|or SQLServer` through the guide, and on other pages, one on guides.tidalmg.com, with this merged we should be sure to update those references. 

Maybe something simply like, _several database engines|vendors_ instead of listing them out each time, now that there are 4(!). just a thought though.

**Pull request merged**